sources/Launcher/test/TestDataInterfaceContainer.cpp

```diff
@@ -16,59 +16,56 @@
 #include <gtest_dynawo.h>
 #include <string>
 
-#ifdef LANG_CXX11
+#ifdef USE_POWSYBL
 #include <thread>
 #endif
 
 namespace DYNAlgorithms {
 
-static boost::shared_ptr<DYN::DataInterface>
-getDataInterface() {
-  std::string iidmFile = "res/IEEE14.iidm";
-  return DYN::DataInterfaceFactory::build(DYN::DataInterfaceFactory::DATAINTERFACE_IIDM, iidmFile, 1);
-}
-
 TEST(TestDataInterfaceContainer, base) {
-  boost::shared_ptr<DYN::DataInterface> dataInterface = getDataInterface();
-  DataInterfaceContainer container(dataInterface);
+  DataInterfaceContainer container("res/IEEE14.iidm", 1);
 
-  ASSERT_FALSE(container.getDataInterface());
+  boost::shared_ptr<DYN::DataInterface> dataInterface = container.getDataInterface();
+#ifdef USE_POWSYBL
+  ASSERT_FALSE(dataInterface);
+#else
+  ASSERT_TRUE(dataInterface);
+#endif
 
   container.initDataInterface(0);
-  // Clone is done : the retrieved data interface is supposed to be different
+
+  // case powsybl: Clone is done
+  // case legacy: IIDM file is read
+  // the retrieved data interface is supposed to be different
   boost::shared_ptr<DYN::DataInterface> dataInterface2 = container.getDataInterface();
   ASSERT_NE(dataInterface2, dataInterface);
 }
 
-#ifdef LANG_CXX11
+#ifdef USE_POWSYBL
```

Comment:
Multi threading should be tested for all cases

Reply:
OK